site2/docs/performance-pulsar-perf.md

```diff
@@ -109,22 +123,34 @@ The following table lists configuration options available for the `pulsar-perf c
 | acks-delay-millis | Set the acknowledgment grouping delay in milliseconds. | 100 ms |
 | auth-params | Set the authentication parameters, whose format is determined by the implementation of the `configure` method in the authentication plugin class, such as "key1:val1,key2:val2" or "{"key1":"val1","key2":"val2"}". | N/A |
 | auth_plugin | Set the authentication plugin class name. | N/A |
+| auto_ack_chunk_q_full | Configure whether to automatically ack for the oldest message in receiver queue if the queue is full. | false |
```

Comment:
underline (_) or hyphen (-)?

Reply:
Please check all occurencies.

Reply:
@Anonymitaet Both `expire_time_incomplete_chunked_messages` and `auto_ack_chunk_q_full` use underline, they come from here
https://github.com/apache/pulsar/blob/f342c5d893527b5daec59cbc67abd3dbfdb02020/pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java#L145-L151

Reply:
OK